### ISSUE TITLE
Demos have broken layout on some browsers due to <hr> sizing

### ISSUE THREAD

**Issue by Haprog** [vaadin/vaadin-radio-button-flow#106]
Some of the demos use `<hr>` elements with `width: 100%; height: 100%` which for some reason seems to look ok in Chrome but makes some of the demos unusable on other browsers (at least on Firefox).

See: https://vaadin.com/components/vaadin-radio-button/java-examples

Here's an example of how it looks on Firefox:

![image](https://user-images.githubusercontent.com/75155/59844873-94902c80-9364-11e9-900e-ca0990751632.png)

Here's how it should look (this is from Chrome):

![image](https://user-images.githubusercontent.com/75155/59844903-a5d93900-9364-11e9-8982-04fcb1cd2d39.png)

In Firefox (and apparently also in Edge and IE11) the `<hr>`s become as tall as the whole demo block.

This seems to be because the demos use this helper method to create the `<hr>`s:
https://github.com/vaadin/vaadin-radio-button-flow/blob/master/vaadin-radio-button-flow-demo/src/main/java/com/vaadin/flow/component/radiobutton/demo/RadioButtonGroupView.java#L338
```java
private Hr getFullSizeHr() {
    Hr hr = new Hr();
    hr.setSizeFull();
    return hr;
}
```

Is there some reason to do this and not just create `new Hr()` without extra styles? I think it should work fine.

**Comment by web-padawan** [comment]
Still valid.

**Comment by shadikhani** [comment]
These are the old demos and in new demos we have not used the `getFullSizeHr()` method.

**Comment by Haprog** [comment]
This was fixed in #114. `getFullSizeHr()` is still being used but only in the test module (for ITs) not for demos.
